src/main/java/com/boomi/flow/services/boomi/mdh/database/FieldMapper.java

```diff
@@ -124,10 +124,9 @@
 
                 if (property.getContentType() == ContentType.List) {
                     mapObject.put(getEntryName(property, universe.getLayout().getModel().getElements()), object);
-                } else if (property.getContentType() == ContentType.Object && object instanceof List) {
+                } else if (property.getContentType() == ContentType.Object && object instanceof HashMap) {
```

Comment:
This should probably be `Map` rather than whatever the current implementation you're using is. That way it'll still work in the case that we start using a different `Map` implementation